### PR TITLE
Adding base.html and logo+ccs file

### LINKED ISSUE #56: Create a base.html file
- [ ] Create file that called base.html, this file should located at \Smarticle\projboard\templates

- [ ] Add navbar, footer and links/scripts that common to all the other html parts 

- [ ] Add some style to the base.html  

- [ ] You can learn more about django-templates and see an example to base.html [here](https://moodle.telhai.ac.il/pluginfile.php/1348555/mod_resource/content/1/Django%20UI%20basics_V2.3_B08.pdf)

### PR DESCRIPTION
**Pull request by yaelnoyman1**
I changed board.html to accept the details I created in base.html.
resolve #56

### CODE REVIEW

**Review by rawadabu — COMMENTED** [review]
I suggest to create a new folder for css files, path `projboard/static/css/`

**Review by rawadabu — COMMENTED** [review]
Same for images [here](https://github.com/redhat-beyond/Smarticle/blob/3f3ab21273c66b90270cd15d6c12bfef305386d7/projboard/static/smarticle/Smarticle_logo_prev_ui.png), I suggest to create a folder for `images`, path `projboard/static/images/`

**Comment by yaelnoyman1** [comment]
> I suggest to create a new folder for css files, path `projboard/static/css/`

**Comment by yaelnoyman1** [comment]
> I suggest to create a new folder for css files, path `projboard/static/css/`

done

**Comment by yaelnoyman1** [comment]
> Same for images [here](https://github.com/redhat-beyond/Smarticle/blob/3f3ab21273c66b90270cd15d6c12bfef305386d7/projboard/static/smarticle/Smarticle_logo_prev_ui.png), I suggest to create a folder for `images`, path `projboard/static/images/`

done

File: projboard/templates/base.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      <br/><br/>
+      <div class="container-fluid mt-3 ps-0 pe-0 h-100 d-inline-block">
+        {% block content %}{% endblock %}
+      </div>
+    </main>
+   
+    <!--footer-->
+    <footer class="bg-dark text-center text-lg-start mt-3">
+          <div class="text-center p-3">
+              <a class="text-light">© 2022: Smarticle </a>
```
Comment (Shalevro2) [inline projboard/templates/base.html:61]: Consider to change the a tag to p tag

File: projboard/templates/base.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                <a class="nav-link" href="#">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Search Article</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">About</a>
+              </li>
+            </ul>
+        </div>
```
Comment (Shalevro2) [inline projboard/templates/base.html]: Please add also a tag to:
Create Article
My Articles

**Review by Yarboa — DISMISSED** [review]
LGTM

**Review by Yarboa — CHANGES_REQUESTED** [review]
Please add at least one client request board.html

**Comment by yaelnoyman1** [comment]
> Please add at least one client request board.html

Hi, I would appreciate an explanation
I didn't really understand what you meant

**Comment by Anas-mhameed** [comment]
> I think @Yarboa want you to add a simple test with the client fixture .

File: projboard/templates/base.html (hunk 1 above)
Comment (yaelnoyman1) [reply projboard/templates/base.html:61]: I prefer to leave it as tag "a" because it adds a line below

File: projboard/templates/base.html (hunk 2 above)
Comment (yaelnoyman1) [reply projboard/templates/base.html]: done

File: projboard/templates/base.html (hunk 1 above)
Comment (Shalevro2) [reply projboard/templates/base.html:61]: Ok, np.

**Comment by yaelnoyman1** [comment]
> Please add at least one client request board.html

done

**Review by finziamit — APPROVED** [review]
LGTM

**Review by Shalevro2 — APPROVED** [review]
LGTM

**Review by rawadabu — APPROVED** [review]

**Review by Yarboa — APPROVED** [review]

**Pull request merged**